lesson06/task3/server/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+)
+
+var name string
+var address string
+
+func handler(w http.ResponseWriter, r *http.Request) {
+
+	if r.URL.Path != "/" {
+		http.Error(w, "404 not found.", http.StatusNotFound)
+		return
+	}
+
+	switch r.Method {
+	case "GET":
+
+		// GET return HTML page
+		http.ServeFile(w, r, "form.html")
+
+	case "POST":
+
+		if err := r.ParseForm(); err != nil {
+			fmt.Fprintf(w, "ParseForm() err: %v", err)
+			return
+		}
+
+		name = r.FormValue("name")
+
+		address = r.FormValue("address")
+
+		fmt.Fprintf(w, "%s from %s\n", name, address)
+		fmt.Printf("%s from %s\n", name, address)
```

Comment:
According to the task description, you need to serve the same `form.html` with the value of your token on the top. Basically on GET you will have the form and on POST you will get the same form, but with the value from a cookie